docs/resources/client/index.md

```diff
@@ -0,0 +1,15 @@
+Resource Packs
+==============
+
+[Resource Packs][respack] allow for the customization of client resources through the `assets` directory. This includes textures, models, sounds, localizations, etc. Forge, and your mod, can also have resource packs. Any user can therefore modify all the textures, models, and other asetts defined within this directory.
+
+### Creating a Resource Pack
+Resource Packs are stored within the `assets` directory within your project's resources.
```

Comment:
It's not _exactly_ correct: the `assets` directory contains the contents of the resource pack. The resource pack itself is defined by the presence of `pack.mcmeta` alongside that folder, and stored in the appropriate location in the game directory: under `resourcepacks` for the user's packs, and under `server-resource-packs` for server-specific packs.

Reply:
While true, we are only talking about in relation to a Forge mod. As such, I believe the documentation on where the game directory stores that information is irrelevant for this.